src/tcgwars/logic/impl/gen4/LegendsAwakened.groovy

```diff
@@ -250,10 +250,10 @@ public enum LegendsAwakened implements LogicCardInfo {
 
               def selected = my.deck.search(max:1,"Select a Deoxys",{it.name == "Deoxys" || (it.name.contains("Deoxys") && it.name.contains("Forme"))}) //Deoxys cannot Form Change into Deoxys ex.
               if (selected) {
-                def tpc = self.topPokemonCard
+                def tpc = new CardList (self.topPokemonCard)
                 selected.moveTo(suppressLog: true, self.cards)
                 tpc.moveTo(suppressLog: true, my.deck)
-                bc "${tpc.name} was swapped with ${selected.name}."
+                bc "${tpc.first().name} was swapped with ${selected.name}."
```

Comment:
Maybe you should use `moveCard` defined in TcgStatics for this instead of creating a new CardList just to use `moveTo`.